src/client/containers/EventsContainer.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+
+import PropTypes from 'prop-types'
+
+import Grid from '@material-ui/core/Grid'
+import { withStyles } from '@material-ui/core/styles'
+
+import EventsView from '../pages/Events/EventsView'
```

Comment:
```suggestion
 import EventsView from 'pages/Events/EventsView'
```

You can use src imports in this project. This will help when we have a deeper component file structure.